scripts/covid_hosp_explore.R

```diff
@@ -1,37 +1,37 @@
 source("scripts/targets-common.R")
 source("scripts/targets-exploration-common.R")
 
+# ================================ GLOBALS =================================
 # These globals are needed by make_forecasts_and_scores (and they need to persist
 # during the actual targets run, since the commands are frozen as expressions).
-hhs_signal <- "confirmed_admissions_covid_1d"
-if (!exists("ref_time_values_")) {
-  # Alternatively you can let slide_forecaster figure out ref_time_values
-  start_date <- as.Date("2023-11-08")
-  end_date <- as.Date("2024-04-24")
-  # end_date <- start_date + 7
-  date_step <- 7L
-  ref_time_values_ <- seq.Date(start_date, end_date, by = date_step)
-}
-time_value_adjust <- 3 # this moves the week marker from Saturday to Wednesday
-
-# Debug mode will replace all forecasters with a fast dummy forecaster. Helps
-# with prototyping the pipeline.
-dummy_mode <- as.logical(Sys.getenv("DUMMY_MODE", FALSE))
-
-# these are for datasets which have some locations with extreme latency; currently that's only nwss
-# note that this has a vector of names first, and then the list of things to
-# exclude, because targets hates named lists and strips their names
-very_latent_locations <- list(list(
-  c("geo_value"),
-  c("la", "ms", "nd")
-))
-# TODO decide on this, nwss is latent in these locations by ~56 days. might be ok
```

Comment:
Weird that these were both here and basically unused? I added it on Nov 13th, I think it may have just not made it all the way to where it was needed.

Reply:
looking at https://delphi-forecasting-reports.netlify.app/covid-notebook-scaled_pop_exogenous it seems like either nwss signal is doing fine however (roughly on par with windowed seasonal) so I guess its a non-issue.

Reply:
Yea, I think it was copy pasta from flu explore, where it's actually used by the aux_data forecasters